src/main/java/com/thevoxelbox/voxelsniper/brush/type/SignOverwriteBrush.java

```diff
@@ -18,20 +19,17 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
 
 public class SignOverwriteBrush extends AbstractBrush {
```

Comment:
Putting this here, but SignOverWriteBrush#saveBufferToFile should save them in a dedicated folder, like `/signoverwrite/`. Else this gets too messy imo.

Reply:
hell yes!